src/components/navigation.js

```diff
@@ -52,12 +53,34 @@ const Navigation = props => {
 	const { logoName, logoNavigateTo } = props
 	const { getTopLevelNavigation, getNonRulesNavigation } = data
 
+	const [isMenuShown, setIsMenuShown] = useState(true)
+	const onMediaQueryChange = matches => {
```

Comment:
```suggestion
	const onMediaQueryChange = matches => setIsMenuShown(!matches.small);
```
And delete the rest of this function.

Reply:
I expect to have to handle for large on a later date. Let us concentrate on the functionality over condensing code.